client/client-multi/client-c/src/kaa/platform-impl/common/kaa_tcp_channel.c

```diff
@@ -1104,17 +1101,18 @@ kaa_error_t kaa_tcp_channel_socket_io_error(kaa_tcp_channel_t *self, kaa_failove
 }
 
 #ifdef KAA_ENCRYPTION
-static kaa_error_t kaa_tcp_channel_encrypt(kaa_tcp_channel_t *self, char **input, size_t *input_size)
+static kaa_error_t kaa_tcp_channel_encrypt(kaa_tcp_channel_t *self, uint8_t **input, size_t *input_size)
```

Comment:
`input` name distorts me. In fact, it is in-out parameter and should be named accordingly